modules/kubernetes/logs/masks/all.river

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
These masks are amazing and absolutely love them. The only suggestion I have is if we can move them out of modules/kubernetes and into something like `modules/logs/mask` since I think people will use these everywhere.

Reply:
@mattdurham  this is where conditional logic would be great.  The problem is this is all done through the `loki.process` component through a `stage.match`.  In K8s we add support for this functionality to opt-in/out through an annotation which is the match stage, which if matched performs the `stage.replace`.  I would prefer to follow DRY and not repeat the module, regex but as of now `loki.process` does not have a stage to include a module.  Any advice?

Reply:
Let me think on that but I think we can add it as is and then figure out a good way to extract it.